changes/v5.0.10-zh.md

```diff
@@ -30,3 +30,7 @@
   `$events/message_dropped`, 如果消息事件是共享订阅产生的，在编码（到 JSON 格式）过程中会失败。
 
 - 修复 HTTP API `/gateways` 的返回状态码，未知 Gateway 名字应返回 `404` 而不是 `400` [#9268](https://github.com/emqx/emqx/pull/9268)。
+
+- Add property `code` to error response for `/authentication/sources/:type`. [9299](https://github.com/emqx/emqx/pull/9299)
```

Comment:
You can ignore the Chinese version, after you make a new PR you could send a message on slack to me, I'm glad to translate it